lib/jira/commands/new.rb

```diff
@@ -2,8 +2,15 @@ module Jira
   class CLI < Thor
 
     desc "new", "Creates a new ticket in JIRA and checks out the git branch"
+    method_option :project, aliases: "-p", type: :string, default: nil, banner: "PROJECT"
+    method_option :components, aliases: "-c", type: :array, default: nil, lazy_default: [], banner: "COMPONENTS"
+    method_option :issuetype, aliases: "-i", type: :string, default: nil, banner: "ISSUETYPE"
+    method_option :parent, aliases: "-p", type: :string, default: nil, lazy_default: "", banner: "PARENT"
```

Comment:
the only issue i have here is that `parent` and `project` both use the `-p` flag. this should be resolved before i merge it in. sorry for the delayed response - been traveling the past few days